app/models/pydantic/creation_options.py

```diff
@@ -127,10 +128,11 @@ def validate_no_data(cls, v, values, **kwargs):
 
     @validator("band_count")
     def validate_band_count(cls, v, values, **kwargs):
+        # Should we really require this or make pixetl more flexible?
```

Comment:
you mean like allow resampling or other non-calc operations across multiple bands?

Reply:
No, I mean that right now we don't require a calc string for a one-band raster and assume that the client wants an output band identical to the input band. But why not do the same for multiple-band inputs? I implemented this (generating a calc string that just passes any input bands through unmodified) in the data API code that creates a raster tile cache because I had to, but it seems to me a reasonable thing to do as a general default. I chickened-out of doing it across the board because it felt bad to eliminate a validator, but if we were to have pixetl pass bands through unmodified by default it wouldn't be validating anything useful. I think I'll leave this check as-is for now, and remove it when pixetl is capable of taking a multi-band input without a calc string.

Reply:
sounds good to me